pandas/tests/api/test_api.py

```diff
@@ -63,7 +63,7 @@ class TestPDApi(Base):
     # top-level functions
     funcs = ['bdate_range', 'concat', 'crosstab', 'cut',
              'date_range', 'interval_range', 'eval',
-             'factorize', 'get_dummies',
+             'factorize', 'get_dummies', 'cdate_range',
```

Comment:
I think cdate_range needs to be in api.rst as well?

Reply:
that's the issue being solved here: it was already added to api.rst in a previous PR, but wasn't actually yet included in the api

Reply:
ok great